rest-api-spec/src/main/resources/rest-api-spec/test/scroll/12_slices.yml

```diff
@@ -104,7 +104,7 @@ setup:
 ---
 "Sliced scroll with invalid arguments":
   - do:
-        catch:  /query_phase_execution_exception.*The number of slices.*index.max_slices_per_scroll/
+        catch:  /illegal_argument_exception.*The number of slices.*index.max_slices_per_scroll/
```

Comment:
we are not testing here the response code. Better to replace with `catch: bad_request`. That way we don't check the message, but we make sure that 400 is returned.